File: Python_Programming_Assignment_1/Assignment1_2.py
# 2. Write a progrm which contains one function named as ChkNum()
#    which accepts one parameter as number. If number is even then
#    it should disply "Even number" otherwise display "Odd number" on console.

# Input : 11                    Output : Odd Number
# Input : 8                     Output : Even Number

# To check number is Even or Odd
def ChkNum(number):
    # if remainder is 0 then number is even
    if number%2 == 0:
        print("Even number")
    # if remainder is nonzero then number is odd
    else:
        print("Odd number")

def main():
    print("Please enter number to check whether its odd or even")
    number = int(input())
    ChkNum(number)

# Starter
if __name__ == "__main__":
    main()

File: Typescript_Assignment/Typescript_Assignment1/Assignment1_2.js
// 1.2 Write a typescript program which contains one function named as Area.
//     That function should calculate area of circle. Accept value of radius
//     from user and return its area. Default value of PI should be 3.14 if 
//     it is not provided by the caller.
// Input : 5
// Output : Area of circle is 78.5
function Area(Radius, PI) {
    if (PI === void 0) { PI = 3.14; }
    return PI * Radius * Radius;
}
console.log("Area of Circle is " + Area(5));
